pkgs/development/tools/misc/hydra-cli/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, pkgconfig, openssl, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  name = "hydra-cli-${version}";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "nlewo";
+    repo = "hydra-cli";
+    rev = "v${version}";
+    sha256 = "1jdlmc45hwblcxs6hvy3gi2dr7qyzs1sg5zr26jrpxrbvqqzrdhc";
+  };
+
+  cargoSha256 = "0dqj2pdqfbgg8r3h2s07p3m9zgl9xl4vislbqs6a0f1ahrczlda5";
+
```

```suggestion
buildInputs = lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
```

Looks like this fails on darwin, please see the checks.